docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md

```diff
@@ -0,0 +1,453 @@
+    |       `mx.nd.contrib.while_loop(cond, func, loop_vars, max_iterations, name)`                |            `mx.npx.while_loop(cond, func, loop_vars, max_iterations, name)`                    |                - moved to `npx` namespace. <br> - Will not support global variables as cond or func's inputs(cond or func's inputs must be in loop_vars)           |
+    |       `mx.nd.contrib.cond(pred, then_func, else_func, inputs, name)`                |            `mx.npx.cond(pred, then_func, else_func, name)`                    |                - moved to `npx` namespace. <br> - users needs to provide the inputs of pred, then_func and else_func as inputs <br> - Will not support global variables as pred, then_func or else_func's inputs(pred, then_func or else_func's inputs must be in inputs)           |
+
+5. Functionalities
+    |                   Legacy Operators               |    NumPy Operators Equivalent    |   Changes  |
+    | ----------------------------------------------------- | ------------------------------ | ------------------- |
+    |       `mx.nd.save(*args, **kwargs)`                |            `mx.npx.savez(*args, **kwargs)`                    |                - moved to `npx` namespace. <br> - Only accept positional arguments, try to flatten the list/dict before feed in          |
+    |       `mx.nd.load(*args, **kwargs)`                |            `mx.npx.load(*args, **kwargs)`                    |                - moved to `npx` namespace.         |
+    |       `mx.nd.waitall()`                |            `mx.npx.waitall()`                    |                - moved to `npx` namespace.         |
+
+Other operator changes are included in [**Appendix/NumPy and NumPy-extension Operators**](#NumPy-and-NumPy-extension-Operators1) 
+
+
+
+### Layers and Blocks
+With deferred compute and tracing being introduced in Gluon2.0, users do not need to interact with symbols any more. There are a lot of changes in building a model with Gluon API, including parameter management and naming, forward pass computing and parameter shape inferencing. We will provide a step-by-step migration guidance on how to build a model with new APIs.
+
+#### Parameter Management and Block Naming
+In Gluon, each Parameter or Block has a name (and prefix). Parameter names are specified by users and Block names can be either specified by users or automatically created. In Gluon 1.x, parameters are accessed via the `params` variable of the `ParameterDict` in `Block`. Users will need to manually use `with self.name_scope():` for children blocks and specify prefix for the top level block. Otherwise, it will lead to wrong name scopes and can return parameters of children blocks that are not in current name scope. An example for initializing the Block and Parameter in Gluon 1.x: 
```

Comment:
```suggestion
In Gluon, each Parameter or Block has a name (and prefix). Parameter names are specified by users and Block names can be either specified by users or automatically created. In Gluon 1.x, parameters are accessed via the `params` variable of the `ParameterDict` in `Block`. Users will need to manually use `with self.name_scope():` for children blocks and specify prefix for the top level block. Otherwise, it will lead to wrong name scopes and can return parameters of children blocks that are not in the current name scope. An example for initializing the Block and Parameter in Gluon 1.x: 
```